Lib/Protocols/IdHL7.pas

```diff
@@ -225,13 +235,16 @@   TIdHL7 = class(TIdBaseComponent)
     FConnectionLimit: Word;
     FIPMask: String;
     FIPRestriction: String;
+    FIPMaskVal : Cardinal;
+    FIPRestrictionVal : Cardinal;
     FIsListener: Boolean;
     FObject: TObject;
     FPreStopped: Boolean;
     FPort: Word;
-    FReconnectDelay: LongWord;
+    FReconnectDelay: Cardinal;

```

Comment:
Why change a bunch of members from LongWord to Cardinal?  Please leave them as LongWord, or use UInt32 if you want to force a 32bit type.

Reply:
Changed as requested
https://github.com/IndySockets/Indy/pull/442/commits/7bd8d29be4574608893391b97fcb794d959042b9